sentry-ruby/lib/sentry/cron/monitor_check_ins.rb

```diff
@@ -44,9 +44,9 @@ def sentry_monitor_check_ins(slug: nil, monitor_config: nil)
           prepend Patch
         end
 
-        def sentry_monitor_slug
+        def sentry_monitor_slug(base_name = name)
```

Comment:
Nit: let's keep calling it `name` as `base_name` could easily be associated `basename` methods in classes like `Pathname` and cause confusion. And IMO using karg will also make the API easier to use & maintain:

```rb
def sentry_monitor_slug(name: self.name)
end
```